### PR TITLE
feat: Add `Node.insert_child()`

### PR DESCRIPTION
**Pull request by JuroOravec**
Note: I wasn't able to figure out whether I need to update the sphinx docs, and how to do that. For one, it seems docs generation requires <= 3.9. And there was also an issue with conflicting dependencies between `alabaster` and `sphinx`. So if it's required, I'm hoping you could help me with that.

@rushter Let me know what you think!

Closes https://github.com/rushter/selectolax/issues/126, closes https://github.com/rushter/selectolax/issues/128

### CODE REVIEW

File: selectolax/lexbor/node.pxi [hunk 1]
```diff
@@ -655,6 +655,57 @@ cdef class LexborNode:
         else:
             raise SelectolaxError("Expected a string or LexborNode instance, but %s found" % type(value).__name__)
 
+    def insert_child(self, str_or_LexborNode value):
```
Comment (JuroOravec) [inline selectolax/lexbor/node.pxi:658]: For simplicity I implemented `insert_child` the same way as surrounding code (e.g. `insert_after`), and just updated the docstring and function calls.

**Comment by rushter** [comment]
Thanks

**Comment by rushter** [comment]
I will fix the docs, the underlying platform changed config format.

**Comment by rushter** [comment]
I've created a new release which will be uploaded in around 3 hours.

**Pull request merged**